code/control/DNRoot.php

```diff
@@ -823,6 +852,28 @@ public function DNProjectList() {
 	}
 
 	/**
+	 * Provide a list of all projects.
+	 *
+	 * @return SS_List
+	 */
+	public function getStarredProjects() {
+		$member = Member::currentUser();
+		if(!$member) {
+			return new ArrayList();
+		}
+
+		$favProjects = $member->StarredProjects();
+
+		$list = new ArrayList();
+		foreach($favProjects as $project) {
+			if($project->CanView($member)) {
```

Comment:
I think this should be `canView`, not `CanView`. PHP is stupid and let's you do whatever case you want.